### PR TITLE
fix: React Native Windows E2E test app

### PR DESCRIPTION
**Pull request by statm**
### Summary

Fix the React Native Windows E2E test app that has been failing ADO CI check `callstack.haul`:
- Re-create E2E app with react-native-windows 0.61, following the guide [here](https://microsoft.github.io/react-native-windows/docs/getting-started).
- Add a task to install VS component `Microsoft.VisualStudio.ComponentGroup.UWP.VC.v141`, which does not come with [the build agent](https://github.com/actions/virtual-environments/blob/master/images/win/Windows2019-Readme.md#visual-studio-2019-enterprise). (This task takes 15~20 minutes, which is not ideal. Hopefully it'll be addressed in the next RNW version.)
- Tweak the test code to give appium longer time (4 sec -> 20 sec) for startup.

Potential future work: test with 'haul start' instead of 'haul bundle' to catch regressions like #734.

### CODE REVIEW

**Comment by statm** [comment]
cc @zamotany I think this is ready for merge 😄

File: fixtures/react_native_windows_e2e_app/babel.config.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module.exports = {
-  presets: [['module:../../packages/haul-babel-preset-react-native', { chakra: true }]],
```
Comment (zamotany) [inline fixtures/react_native_windows_e2e_app/babel.config.js:2]: Is there any specific reason why switching to Metro's Babel preset??

Comment (statm) [reply fixtures/react_native_windows_e2e_app/babel.config.js:2]: It comes with default RNW project template. Let me test Haul preset and fix it.

Comment (statm) [reply fixtures/react_native_windows_e2e_app/babel.config.js]: Fixed. Thanks for the catch :)

**Comment by thymikee** [comment]
Lovely contribution <3

**Comment by zamotany** [comment]
Thanks again for the work @statm 💯 👍

**Pull request merged**